extensions/amp-app-banner/1.0/component/ios.js

```diff
@@ -0,0 +1,99 @@
+import {logger} from '#preact/logger';
+import {docInfo} from '#preact/utils/docInfo';
+import {platformUtils} from '#preact/utils/platform';
+import {urlUtils} from '#preact/utils/url';
+
+import {openWindowDialog} from '../../../../src/open-window-dialog';
+
+const OPEN_LINK_TIMEOUT = 1500;
+
+/**
+ * @return {{
+ *    openOrInstall: function(): void,
+ *    installAppUrl: string,
+ *    openInAppUrl: string,
+ * }|null}
+ */
+export function getIOSAppInfo() {
+  const canShowBuiltinBanner = platformUtils.isSafari();
+  if (canShowBuiltinBanner) {
+    logger.info(
+      'BENTO-APP-BANNER',
+      'Not rendering bento-app-banner:',
+      'Browser supports builtin banners.'
+    );
+    return null;
+  }
+
+  const metaContent = docInfo.getMetaByName('apple-itunes-app');
```

Comment:
Are we planning on a follow up to pass these as context?

Reply:
Yes, it will be addressed in https://github.com/ampproject/amphtml/issues/37465